src/content/ja/2019/page-weight.md

```diff
@@ -0,0 +1,775 @@
+  <figcaption id="fig10-caption">図10. GIFファイルサイズの累積分布関数。</figcaption>
+</figure>
+
+以下の表は、これらの小さな画像をデータセットから削除するための2つの異なるアプローチを示しています。最初の方法は、ファイルサイズが100バイトを超える画像に基づいており、2番目はファイルサイズが1024バイトを超える画像に基づいています。
+
+#### 画像の画像形式ごとのファイルサイズ> 100バイト
+
+<figure>
+  <table>
+    <tr>
+      <th>パーセンタイル</th>
+      <th>GIF (KB)</th>
+      <th>ICO (KB)</th>
+      <th>JPG (KB)</th>
+      <th>PNG (KB)</th>
+      <th>SVG (KB)</th>
+      <th>WEBP (KB)</th>
+    </tr>
+    <tr>
+      <td>10</td>
+      <td>0.27</td>
+      <td>0.31</td>
+      <td>3.08</td>
+      <td>0.4</td>
+      <td>0.28</td>
+      <td>2.1</td>
+    </tr>
+    <tr>
+      <td>25</td>
+      <td>0.75</td>
+      <td>0.6</td>
+      <td>7.7</td>
+      <td>1.17</td>
+      <td>0.46</td>
+      <td>4.4</td>
+    </tr>
+    <tr>
+      <td>50</td>
+      <td>2.14</td>
+      <td>1.12</td>
+      <td>20.47</td>
+      <td>4.35</td>
+      <td>0.95</td>
+      <td>11.54</td>
+    </tr>
+    <tr>
+      <td>75</td>
+      <td>7.34</td>
+      <td>4.19</td>
+      <td>61.13</td>
+      <td>21.39</td>
+      <td>2.67</td>
+      <td>31.21</td>
+    </tr>
+    <tr>
+      <td>90</td>
+      <td>35</td>
+      <td>14.73</td>
+      <td>155.46</td>
+      <td>91.02</td>
+      <td>8.26</td>
+      <td>76.43</td>
+    </tr>
+  </table>
+  <figcaption>図11. 100バイトを超える画像の画像形式ごとのファイルサイズ。</figcaption>
+</figure>
+
+#### 画像の画像形式ごとのファイルサイズ > 1024バイト
+
+<figure>
+  <table>
+    <tr>
+      <th>パーセンタイル</th>
+      <th>GIF (KB)</th>
+      <th>ICO (KB)</th>
+      <th>JPG (KB)</th>
+      <th>PNG (KB)</th>
+      <th>SVG (KB)</th>
+      <th>WEBP (KB)</th>
+    </tr>
+    <tr>
+      <td>10</td>
+      <td>1.28</td>
+      <td>1.12</td>
+      <td>3.4</td>
+      <td>1.5</td>
+      <td>1.2</td>
+      <td>3.08</td>
+    </tr>
+    <tr>
+      <td>25</td>
+      <td>1.9</td>
+      <td>1.12</td>
+      <td>8.21</td>
+      <td>2.88</td>
+      <td>1.52</td>
+      <td>5</td>
+    </tr>
+    <tr>
+      <td>50</td>
+      <td>4.01</td>
+      <td>2.49</td>
+      <td>21.19</td>
+      <td>8.33</td>
+      <td>2.81</td>
+      <td>12.52</td>
+    </tr>
+    <tr>
+      <td>75</td>
+      <td>11.92</td>
+      <td>7.87</td>
+      <td>62.54</td>
+      <td>33.17</td>
+      <td>6.88</td>
+      <td>32.83</td>
+    </tr>
+    <tr>
+      <td>90</td>
+      <td>67.15</td>
+      <td>22.13</td>
+      <td>157.96</td>
+      <td>127.15</td>
+      <td>19.06</td>
+      <td>79.53</td>
+    </tr>
+  </table>
+  <figcaption>図12. 1024バイトを超える画像の画像形式ごとのファイルサイズ。</figcaption>
+</figure>
+
+JPEG画像に比べてPNG画像のファイルサイズが小さいことは驚くべきことです。 JPEGは[非可逆圧縮](https://en.wikipedia.org/wiki/Lossy_compression)を使用します。非可逆圧縮によりデータが失われるため、ファイルサイズを小さくできます。一方、PNGは[可逆圧縮](https://en.wikipedia.org/wiki/Lossless_compression)を使用します。これによりデータが失われることはありません。これにより、より高品質で大きな画像が生成されます。ただし、このファイルサイズの違いはエンコーディングと圧縮の違いではなく、透過性のサポートによるアイコンのグラフィックのPNGの人気を反映している可能性があります。
```

Comment:
Looks like there are Japanese versions of these Wikipedia pages so should we use them?

```suggestion
JPEG画像に比べてPNG画像のファイルサイズが小さいことは驚くべきことです。 JPEGは[非可逆圧縮](https://ja.wikipedia.org/wiki/%E9%9D%9E%E5%8F%AF%E9%80%86%E5%9C%A7%E7%B8%AE)を使用します。非可逆圧縮によりデータが失われるため、ファイルサイズを小さくできます。一方、PNGは[可逆圧縮](https://ja.wikipedia.org/wiki/%E5%8F%AF%E9%80%86%E5%9C%A7%E7%B8%AE)を使用します。これによりデータが失われることはありません。これにより、より高品質で大きな画像が生成されます。ただし、このファイルサイズの違いはエンコーディングと圧縮の違いではなく、透過性のサポートによるアイコンのグラフィックのPNGの人気を反映している可能性があります。
```

Reply:
Thanks review, I fix.